src/components/FormInput.js

```diff
@@ -25,6 +26,7 @@ module.exports = React.createClass({
 	getDefaultProps() {
 		return {
 			type: 'text',
+			autoComplete: 'off'
```

Comment:
I would not set to `off` by default. That is not how browsers treat it if unset. Just keep it unset and let browser decide